src/hotspot/os/linux/gc/z/zMountPoint_linux.cpp

```diff
@@ -88,14 +90,15 @@ void ZMountPoint::get_mountpoints(const char* filesystem, ZArray<char*>* mountpo
     }
   }
 
-  os::free(line);
+  // readline will return malloced memory. Need raw ::free, not os::free.
```

Comment:
You mean `getline`?

Reply:
> You mean `getline`?

Yes.
Sorry, it's too late.

Shall we open another issue to fix it?